TestModels/Constructor/runtimes/rust/src/conversions/simple_constructor_config/_simple_constructor_config.rs

```diff
@@ -0,0 +1,149 @@
+// Code generated by software.amazon.smithy.rust.codegen.smithy-rs. DO NOT EDIT.
+#[allow(dead_code)]
+
+pub fn to_dafny(
+    config: crate::config::Config,
+) -> ::std::rc::Rc<
+    ::constructor_dafny::_simple_dconstructor_dinternaldafny_dtypes::SimpleConstructorConfig,
+> {
+    ::std::rc::Rc::new(
+        ::constructor_dafny::_simple_dconstructor_dinternaldafny_dtypes::SimpleConstructorConfig::SimpleConstructorConfig { blobValue: blob_to_dafny(config.blob_value), booleanValue: boolean_to_dafny(config.boolean_value), stringValue: string_to_dafny(config.string_value), integerValue: integer_to_dafny(config.integer_value), longValue: long_to_dafny(config.long_value) })
+}
+
+fn blob_to_dafny(
```

Comment:
These look like duplicates of the helper methods in `dafny_standard_library`, e.g. https://github.com/smithy-lang/smithy-dafny/blob/main-1.x/TestModels/dafny-dependencies/StandardLibrary/runtimes/rust/src/conversion.rs#L87. Can you use those instead?

Reply:
removed all of these except for `Blob` since we're reverting to `Vec<u8>`